fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchPerformaceTest.java

```diff
@@ -46,7 +46,7 @@
     private String observationId;
```

Comment:
Rename this file 
SearchPerformaceTest.java 
SearchPerformanceTest.java

Reply:
ha, typo error. will fix